### ISSUE TITLE
[Logging tab] Missing keyboard navigation in table

### ISSUE THREAD

**Issue by julemand101** [flutter/devtools#5987]
In the "Logging" tab, you can't use the keyboard arrows up/down to go though the list.

Without being sure, it does look like this is a general issue with all tables in DevTools. I can't at least seem to find any table where I can use the keyboard to navigate it.

Tested with: DevTools version 2.23.1 in both Firefox and Edge.

**Comment by kenzieschmoll** [comment]
Closing as a duplicate of https://github.com/flutter/devtools/issues/4462
